jwt/options.go

```diff
@@ -167,7 +167,7 @@ func WithKeySet(set jwk.Set, options ...interface{}) ParseOption {
 // WithIssuer specifies that expected issuer value. If not specified,
 // the value of issuer is not verified at all.
 func WithIssuer(s string) ValidateOption {
-	return WithValidator(ClaimValueIs(IssuerKey, s))
+	return WithValidator(IssuerClaimValueIs(s))
```

Comment:
I don't think it's correct to add an exported function here.
The public API to check `iss` is already there (`WithIssuer`). Its internals need not be public.